server/Dockerfile

```
@@ -206,6 +206,17 @@ RUN ARCH=$(uname -m) && \
 # Install Tailscale CLI (for local dev SSH to tailnet devices)
 RUN curl -fsSL https://tailscale.com/install.sh | sh
 
+# Install Fly.io CLI (flyctl) for Fly.io connector
+RUN ARCH=$(uname -m) && \
```

Hadolint (2.14.0)

[warning] FLY_ARCH appears unused. Verify use (or export if used externally).

(SC2034)